Applications/DataExplorer/VtkVis/VtkAlgorithmProperties.cpp

```diff
@@ -22,7 +22,7 @@
 #include <vtkTexture.h>
 
 #include "VtkColorLookupTable.h"
-#include "FileIO/XmlIO/Qt/XmlLutReader.h"
+#include "XmlIO/Qt/XmlLutReader.h"
```

Comment:
I won't correct it. DataExplorer doesn't use a full path at many places and those I corrected are exceptions using a full path.